library/common/extensions/filters/http/test_kv_store/config.cc

```diff
@@ -0,0 +1,29 @@
+#include "library/common/extensions/filters/http/test_kv_store/config.h"
+
+#include "library/common/extensions/filters/http/test_kv_store/filter.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace TestKeyValueStore {
```

Comment:
I would have expected the name of this to be something like PlatformKeyValueStore. Is this filter intended to be used in production? Maybe it's just test-only?

Reply:
Your presumption is correct - this is a test-only filter that exist alongside some other test-only filters in an entirely unintuitive location. Ideally, these should be moved elsewhere. I can certainly add a clarifying comment.